packages/flutter/lib/src/material/material_localizations.dart

```diff
@@ -160,6 +160,41 @@ abstract class MaterialLocalizations {
   /// as a hint text in the text field.
   String get searchFieldLabel;
 
+  /// Label indicating ascending sort order, used in onTapHint for column headings
+  /// in a table.
+  String get sortOrderAscendingLabel;
+
+  /// Label indicating descending sort order, used in onTapHint for column headings
+  /// in a table.
+  String get sortOrderDescendingLabel;
+
+  /// Used as onTapHint for an individual cell in a table, indicating tapping
```

Comment:
keep things general, i.e. avoid mentioning specific API here and above

Reply:
Thanks! Done.